src/azure-cli/azure/cli/command_modules/sql/custom.py

```diff
@@ -2628,6 +2628,32 @@ def managed_db_restore(
         parameters=kwargs)
 
 
+def managed_db_restore_ltr_backup(
+        cmd,
+        client,
+        long_term_retention_backup_resource_id,
+        target_managed_database_name,
+        target_managed_instance_name,
+        target_resource_group_name,
+        **kwargs):
+    '''
+    Restores a managed database from a long term retention backup.
+    '''
+    kwargs['location'] = _get_managed_instance_location(
+        cmd.cli_ctx,
+        managed_instance_name=target_managed_instance_name,
+        resource_group_name=target_resource_group_name)
+
+    kwargs['create_mode'] = CreateMode.restore_long_term_retention_backup.value
```

Comment:
Should we expose the restore mode to users? Currently you are hard-code the mode with longtermretention, what if point in time restore?

Reply:
Sure; it looked like the command "sql midb restore" already hard-coded PITR, so I just did something similar with "sql midb ltr restore"